modules/ROOT/pages/database-administration/standard-databases/errors.adoc

```diff
@@ -60,56 +60,27 @@ A database management operation may fail for a number of reasons.
 For example, if the file system instance has incorrect permissions, or Neo4j itself is misconfigured.
 As a result, the contents of the `statusMessage` column in the `SHOW DATABASE` query results may vary significantly.
 
-However, databases may only be in one of a select number of states:
+However, databases may only be in one of a select number of states (see xref::database-administration/standard-databases/manage-databases.adoc#database-states[Database states] for more information):
 
-[options="header" cols="m,a"]
-|===
-| State
-| Description
-
-| online
-| The database is running.
-
-| offline
-| The database is not running.
-If the `statusMessage` column is filled, the database is not running because of a problem.
-
-| starting
-| The database is not running, but is about to.
-
-| stopping
-| The database is not running anymore, but still has not stopped completely.
-No offline operations (e.g. `load`/`dump`) can be performed yet.
-
-| store copying
-| The database is currently being updated from another instance of Neo4j.
 
-| initial
-| The database has not yet been created.
-
-| dirty
-| This state implies an error has occurred.
-The database's underlying store files may be invalid.
-For more information, consult the `statusMessage` column or the server's logs.
-
-| quarantined
-| The database is effectively stopped and its state may not be changed until no longer quarantined.
-For more information, consult the `statusMessage` column or the server's logs.
-
-| unknown
-| This instance of Neo4j doesn’t know the state of this database.
-
-| dropped
-| The database has been deleted.
-|===
+* `online`
+* `offline`
+* `starting`
+* `stopping`
+* `store copying`
+* `initial`
+* `deallocating`
+* `dirty`
+* `quarantined`
+* `unknown`
 
 Most often, when a database management operation fails, Neo4j attempts to transition the database in question to the `offline` state.
 If the system is certain that no store files have yet been created, it transitions the database to `initial` instead.
 Similarly, if the system suspects that the store files underlying the database are invalid (incomplete, partially deleted, or corrupt), then it transitions the database to `dirty`.
 
 [NOTE]
 ====
-While `dropped` is a valid database state, it is only transiently observable, as database records are removed from `SHOW DATABASE` results once the `DROP` operation is complete.
+Additionally, `dropped` is also a valid database state, but it is only transiently observable, as database records are removed from `SHOW DATABASE` results once the `DROP` operation is complete.
```

Comment:
Not even, that, today in 5.x there is no code-path which could show you `dropped`/

Reply:
`dropped` is now dropped 👍